### PR TITLE
Removed change frequency and priority from sitemap

### LINKED ISSUE #9765: Remove updatefreq and priority from sitemap
[Reliable info](https://www.seroundtable.com/google-priority-change-frequency-xml-sitemap-20273.html) says that change frequency and priority attributes of [XML sitemaps](https://demo.ghost.io/sitemap.xml) haven't been used for a while. The attributes have also generally [been removed](https://kb.yoast.com/kb/can-i-change-the-sitemap-priority/) in other software.

So it seems pretty safe to drop them from Ghost's XML sitemaps at this point!

### PR DESCRIPTION
**Pull request by damanm24**
Removed change freq and priority from sitemap and tests

closes #9765
Google says that change frequency and priority are not important when search engines crawl through pages. Therefore, these two properties are unimportant and removed from Ghost. [Reference](https://www.seroundtable.com/google-priority-change-frequency-xml-sitemap-20273.html)

### CODE REVIEW

**Comment by damanm24** [comment]
That's odd on my local machine all the tests pass, I am unsure as to why they fail when run through Travis CI. Any ideas?

**Comment by freezboltz** [comment]
Same thing happen to me
 I think some extra test is going on here

**Comment by kirrg001** [comment]
Tests sometimes fail randomly in Travis.

File: core/server/data/xml/sitemap/base-generator.js [hunk 1]
```diff
@@ -65,10 +64,6 @@ class BaseSiteMapGenerator {
         this.lastModified = Date.now();
     }
 
-    getPriorityForDatum() {
```
Comment (kirrg001) [inline core/server/data/xml/sitemap/base-generator.js]: There are some more occurrences of `getPriorityForDatum` if you do search e.g. in tag-generator.

**Review by kirrg001 — CHANGES_REQUESTED** [review]
Thanks for the PR. Looks good to me. Just left one comment - there are some more functions to cleanup.

**Comment by kirrg001** [comment]
@damanm24 ping :)

**Comment by kirrg001** [comment]
I would like to take over this PR and finish it up later :) Let me know if you are around today and if you would like to fix the comments on your own.

**Comment by damanm24** [comment]
> 
> 
> I would like to take over this PR and finish it up later :) Let me know if you are around today and if you would like to fix the comments on your own.

Yes please take over the PR. I don't have the time to get to it. Thanks.

**Review by kirrg001 — APPROVED** [review]
Updated the PR 👍

**Pull request merged**